src/components/Button.tsx

```diff
@@ -151,12 +151,12 @@ function getColors(type: BtnTypes, disabled: boolean | undefined) {
       backgroundColor = disabled ? colors.goldFaint : colors.goldUI
       break
     case BtnTypes.ONBOARDING:
-      textColor = colors.onboardingBlue
-      backgroundColor = colors.onboardingLightBlue
+      textColor = colors.light
+      backgroundColor = colors.onboardingGreen
```

Comment:
It looks like this change will impact several other pages and components such as: `ImportWallet.tsx` and `VerificationStartScreen.tsx` is this expected?

Reply:
I think we can safely merge all of the work into main by Feb 22nd. It says in the AC to ensure that its all in the same release.

Good idea though for other larger feature changes / redesigns like this @jophish